### PR TITLE
Feature add aliveness api

### PR DESCRIPTION
**Pull request by Ithrael**
add aliveness-test api

### CODE REVIEW

File: config.example.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
     "rabbit_url": "http://127.0.0.1:15672",
     "rabbit_user": "guest",
     "rabbit_pass": "guest",
-    "publish_port": "9419",
+    "publish_port": "9149",
```
Comment (kbudde) [inline config.example.json:5]: Can you please revert this change.

**Review by kbudde — CHANGES_REQUESTED** [review]
Thank you for your contribution.
There is just a small change needed.

File: config.example.json (hunk 1 above)
Comment (Ithrael) [reply config.example.json:5]: sorry, I didn't get the message in time。I'll commit PR again

Comment (Ithrael) [reply config.example.json:5]: https://github.com/kbudde/rabbitmq_exporter/pull/254
